srcpkgs/thrift/template

```diff
@@ -1,24 +1,30 @@
 # Template file for 'thrift'
 pkgname=thrift
-version=0.13.0
-revision=6
-build_style=gnu-configure
-configure_args="--without-python"
-makedepends="boost-devel openssl-devel"
+version=0.17.0
+revision=1
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS=ON
+ -DWITH_AS3=OFF
+ -DWITH_JAVA=OFF
+ -DWITH_JAVASCRIPT=OFF
+ -DWITH_NODEJS=OFF
+ -DWITH_PYTHON=OFF"
```

Comment:
```suggestion
configure_args="-DBUILD_SHARED_LIBS=ON -DWITH_AS3=OFF -DWITH_JAVA=OFF
 -DWITH_JAVASCRIPT=OFF -DWITH_NODEJS=OFF -DWITH_PYTHON=OFF"
```

I do not think every option should be splattered one per line; correct me if I am wrong though.